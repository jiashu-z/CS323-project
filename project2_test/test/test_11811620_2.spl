struct A
{
  int num;
};

struct A func1(int a1, float b1, char c1, struct A d1){
  return a1;
}

struct B func1(int a1, float b2, char c2, struct A d2){
  return d2;
}

struct A func3(int a3, float b3, char c3, struct A d3){
  return e3;
}

int func4(int a4, float b4, char c4, struct A d4){
  func2(a4, b4, c4, d4);
  func3(a4, b4, c4, d4);
}

int test_11811620_2()
{
  int a = 1 + e;
  float b;
  char c;
  struct A c;
  func2(a, b, c, f);
  func1(a, b, c, f);
}