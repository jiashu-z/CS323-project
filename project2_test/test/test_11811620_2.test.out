Error type 8 at Line 7: incompatiable return type
Error type 16 at Line 10: undefined struct type: B
Error type 3 at Line 10: redefine variable: a1
Error type 4 at Line 10: redefine function: func1
Error type 1 at Line 15: undefined variable: e3
Error type 2 at Line 19: undefined function: func2
Error type 17 at Line 18: missing return statement
Error type 1 at Line 25: undefined variable: e
Error type 3 at Line 28: redefine variable: c
Error type 1 at Line 29: undefined variable: f
Error type 2 at Line 29: undefined function: func2
Error type 1 at Line 30: undefined variable: f
Error type 9 at Line 30: invalid argument type for func1. expected: struct, actual: UNKNOWN
